docs/human-interface-guidelines.md

```diff
@@ -384,10 +384,10 @@ Make both the primary and secondary text selectable. This makes it easy for the
 
 ![](/images/docs/human-interface-guidelines/dialogs/button-order.png)
 
-* All dialogs should contain an affirmative button that performs the action suggested in the primary text. This button goes on the far right side of the window.
-* For dialogs that are displayed in response to user action (such as "Quit"), provide a "Cancel" button directly to the left of the affirmative button.
-* If your dialog has alternative actions, list them to the left of the "Cancel" button.
-* If you wish your dialog to contain a "Help" button, this should be placed to the far left of the window.
+* All dialogs should contain an affirmative button that performs the action suggested in the primary text. This button should be aligned to the very end of the dialog.
+* For dialogs that are displayed in response to user action (such as "Quit"), provide a "Cancel" button directly before the affirmative button.
+* If your dialog has alternative actions, list them before the "Cancel" button.
+* If your dialog has incidental actions (actions that do not close the dialog such as "Help"), this should be aligned to the start of the dialog.
```

Comment:
Sentence contains mixed number - "incidental actions" ... "this should be aligned".

Reply:
Agree, should likely be "...**these** should be aligned..."